music-ecom/src/Components/Navbar/Navbar.js

```diff
@@ -1,32 +1,72 @@
 import React from "react";
 import "./Navbar.css";
 import { FaSearch, FaHeart, FaShoppingCart } from "react-icons/fa";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
+import { useWishList } from "../../Context/WishListContext";
+import { useCart } from "../../Context/CartContext";
+import { useAuth } from "../../Context/AuthContext";
```

Comment:
You may create a separate file as context and import everything from there

Reply:
Will do that.